gravitl/netmaker-ui
# fix(GRA-1032): rename ipv6 regex, remove unused regex

depends on backend changes for story GRA-1032

TESTING STEPS
1. join a dual stack (ipv6) network
2. edit a node, set ipv6 address
3.  check that save is successful. check also for ipv4
